### ISSUE TITLE
Unwanted whitespace added between closing HTML tag and period

### ISSUE THREAD

**Issue by sbleon** [jcberquist/commandbox-cfformat#124]
cfformat is adding an awkward whitespace before the period at the end of a sentence when the last word of the sentence is surrounded by HTML tags. Specifically, the whitespace between the `</strong>` and the period.

Input:
```
<cfcomponent>
  <cffunction name='blerg'>
    <cfsavecontent variable='foo'>
      <p>Your #messageParams['clientName']# parking manager would like to know about your interest in using,
      or continuing to use, <strong>#local.parkingSpot#</strong>. Your vote will help us gauge demand and
      usage of the resource, and make an informed decision about how to best allocate the space.</p>
    </cfsavecontent>
  </cffunction>
</cfcomponent>
```

Output:
```
<cfcomponent>
  <cffunction name='blerg'>
    <cfsavecontent variable='foo'>
      <p>
        Your #messageParams['clientName']# parking manager would like to know about your interest in using,
        or continuing to use, <strong>#local.parkingSpot#</strong>
        . Your vote will help us gauge demand and
        usage of the resource, and make an informed decision about how to best allocate the space.
      </p>
    </cfsavecontent>
  </cffunction>
</cfcomponent>
```

Config:
```
{
  "alignment.consecutive.assignments": false,
  "alignment.consecutive.params": false,
  "alignment.consecutive.properties": false,
  "array.empty_padding": false,
  "array.multiline.comma_dangle": false,
  "array.multiline.element_count": 4,
  "array.multiline.leading_comma": false,
  "array.multiline.leading_comma.padding": true,
  "array.multiline.min_length": 40,
  "array.padding": false,
  "binary_operators.padding": true,
  "brackets.padding": false,
  "comment.asterisks": "align",
  "for_loop_semicolons.padding": true,
  "function_anonymous.empty_padding": false,
  "function_anonymous.group_to_block_spacing": "spaced",
  "function_anonymous.multiline.comma_dangle": false,
  "function_anonymous.multiline.element_count": 4,
  "function_anonymous.multiline.leading_comma": false,
  "function_anonymous.multiline.leading_comma.padding": true,
  "function_anonymous.multiline.min_length": 40,
  "function_anonymous.padding": false,
  "function_anonymous.spacing_to_group": false,
  "function_call.casing.builtin": "pascal",
  "function_call.casing.userdefined": "camel",
  "function_call.empty_padding": false,
  "function_call.multiline.comma_dangle": false,
  "function_call.multiline.element_count": 4,
  "function_call.multiline.leading_comma": false,
  "function_call.multiline.leading_comma.padding": true,
  "function_call.multiline.min_length": 40,
  "function_call.padding": false,
  "function_declaration.empty_padding": false,
  "function_declaration.group_to_block_spacing": "spaced",
  "function_declaration.multiline.comma_dangle": false,
  "function_declaration.multiline.element_count": 4,
  "function_declaration.multiline.leading_comma": false,
  "function_declaration.multiline.leading_comma.padding": true,
  "function_declaration.multiline.min_length": 40,
  "function_declaration.padding": false,
  "function_declaration.spacing_to_group": true,
  "indent_size": 2,
  "keywords.block_to_keyword_spacing": "spaced",
  "keywords.empty_group_spacing": false,
  "keywords.group_to_block_spacing": "spaced",
  "keywords.padding_inside_group": false,
  "keywords.spacing_to_block": "spaced",
  "keywords.spacing_to_group": true,
  "max_columns": 120,
  "metadata.multiline.element_count": 4,
  "metadata.multiline.min_length": 40,
  "method_call.chain.multiline": 3,
  "newline": "os",
  "parentheses.padding": false,
  "property.multiline.element_count": 4,
  "property.multiline.min_length": 40,
  "strings.attributes.quote": "single",
  "strings.convertNestedQuotes": false,
  "strings.quote": "single",
  "struct.empty_padding": false,
  "struct.multiline.comma_dangle": false,
  "struct.multiline.element_count": 4,
  "struct.multiline.leading_comma": false,
  "struct.multiline.leading_comma.padding": true,
  "struct.multiline.min_length": 40,
  "struct.padding": true,
  "struct.quote_keys": false,
  "struct.separator": ": ",
  "tab_indent": false,
  "tags.lowercase": true
}
```

**Comment by sbleon** [comment]
Once option that has been suggested is providing a configuration option to skip formatting inside `cfsavecontent`s.

**Comment by jcberquist** [comment]
I've added a fix for this in `v0.16.5` - it is a pretty narrow fix, but it will take care of your specific situation, I believe. More work might need to be done down the road.

**Comment by sbleon** [comment]
Thanks so much, John!
